client/src/unifyfs.c

```diff
@@ -818,15 +818,18 @@ int unifyfs_fid_update_file_meta(int fid, unifyfs_file_attr_t* gfattr)
     return UNIFYFS_FAILURE;
 }
 
-int unifyfs_set_global_file_meta(int gfid, unifyfs_file_attr_t* gfattr)
```

Comment:
Can you add a variation of the `invoke_client_metaset_rpc()` comment here to better explain the `create` value, and what this function does?

Reply:
Alternatively, you could split this into two functions and get rid of the create field:
```
int unifyfs_set_global_file_meta(int gfid, unifyfs_file_attr_t* gfattr)    /* create=0 */
int unifyfs_update_global_file_meta(int gfid, unifyfs_file_attr_t* gfattr) /* create=1 */
```

Reply:
Took the first option of adding a comment.  I imagine we'll want to turn this into a bit field or something to allow the caller to pick and choose exactly which subset of fields should be set.  But that's enough work for a separate PR.